podcasts/Sharing/ShareImageView.swift

```diff
@@ -0,0 +1,108 @@
+import SwiftUI
+import Kingfisher
+
+struct ShareImageInfo {
+    let name: String
+    let title: String
+    let description: String
+    let artwork: URL
+    let gradient: Gradient
+}
+
+enum ShareImageStyle: CaseIterable {
+    case large
+    case medium
+    case small
+
+    var tabString: String {
+        switch self {
+        case .large:
+            return "large"
+        case .medium:
+            return "medium"
+        case .small:
+            return "small"
+        }
+    }
+}
+
+struct ShareImageView: View {
+
+    let info: ShareImageInfo
+    let style: ShareImageStyle
+
+    var body: some View {
+        ZStack {
+            LinearGradient(gradient: info.gradient, startPoint: .top, endPoint: .bottom)
+            Color.black.opacity(0.2)
+            switch style {
+            case .large:
+                VStack(spacing: 32) {
+                    image()
+                        .frame(width: 200, height: 200)
+                    text()
+                    PocketCastsLogoPill()
+                }
+                .padding(24)
+                .frame(width: 292, height: 438)
+            case .medium:
+                VStack(spacing: 24) {
+                    image()
+                        .frame(width: 120, height: 120)
+                    text()
+                        .frame(alignment: .leading)
+                }
+                .padding(24)
+                .frame(width: 292, height: 293)
+            case .small:
+                HStack(spacing: 18) {
+                    image()
+                        .frame(width: 120, height: 120)
+                    text(alignment: .leading, textAlignment: .leading, lineLimit: 3)
+                }
+                .padding(24)
+                .frame(width: 324, height: 169)
+            }
+        }
+        .fixedSize()
+        .clipShape(RoundedRectangle(cornerRadius: 12))
+    }
+
+    @ViewBuilder func image() -> some View {
+        KFImage(info.artwork)
```

Comment:
I believe we should use episode artwork if available (and enabled). Not necessarily needs to be addressed here (we can follow-up on a subsequent PR)

Reply:
👍 Good catch! I've added this as a task to the project thread so I don't forget this.